packages/ses/package.json

```diff
@@ -37,14 +37,12 @@
     ".": {
       "types": "./types.d.ts",
       "import": "./index.js",
-      "require": "./dist/ses.cjs",
-      "types": "./types.d.ts"
```

Comment:
The "types" changes seem unrelated

Reply:
It is. This was a byproduct of running `./scripts/set-versions.sh`, which went ahead and removed the duplicate key from this `package.json`.

Reply:
(and I’m retaining an individual commit for this change so it’s easy to identify/remove/move. I’ll accommodate your preference. Mine is to preserve the commit and save time.)

Reply:
Oh I missed the duplication!